src/Microsoft.Framework.Runtime/NuGet/SemanticVersion.cs

```diff
@@ -13,14 +13,12 @@ namespace NuGet
     /// </summary>
     public sealed class SemanticVersion : IComparable, IComparable<SemanticVersion>, IEquatable<SemanticVersion>
     {
-        private readonly string _originalString;
```

Comment:
No, it's not a random change. The original string is consumed by external code so I expose it as a property.